tests/testthat/test-apply_delay.R

```diff
@@ -206,3 +207,45 @@ test_that("apply_delay function works the same as the more verbose for loop", {
 
   expect_identical(result, expectation2)
 })
+
+test_that("apply_delay works with ragged reporting triangles", {
+  ragged_triangle <- matrix(
```

Comment:
I am missing something, shouldn't this be wide? As written its not testing the raggedness